### PR TITLE
fix #36387 pi-shop.hu

### LINKED ISSUE #36387: pi-shop.hu
### Issue URL (Annoyance)
[https://www.pi-shop.hu/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.pi-shop.hu%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | And 10
AdGuard version: | v3.1.77η (1.4.78cl)
Browser: | Chrome
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Block third-party Authorization header, Self-destructing first-party cookies (4320), Self-destructing third-party cookies (180), Disable cache for third-party requests, Block WebRTC, Block Location API, Hide your Referrer from third-parties (), Hide your Referrer from third-parties, Hide your User-Agent (), Hide your IP address (209.141.53.254)
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | enabled
Filters: | AdGuard Mobile Ads,<br>AdGuard Annoyances,<br>AdGuard Base,<br>hufilter,<br>Malware Domains,<br>Korean Adblock List,<br>Fanboy's Anti-thirdparty Fonts,<br>NoCoin Filter List,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard Japanese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/36387

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -298,7 +298,7 @@ msnbc.com,nbcnews.com###cx_bottom_banner
 pokemongolive.com##.nl-cpb
 openedition.org###privacynotice
 eartharxiv.org##div[class^="__cookie-banner__"]
-netenpiac.hu,profikisgep.hu,majorsoft.eu###firstLogNanobar
+pi-shop.hu,netenpiac.hu,profikisgep.hu,majorsoft.eu###firstLogNanobar
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/cookies_specific.txt]: Please hide the element with #$# rules (when scrolling down you have white bar at top).

```
pi-shop.hu#$##mobile-nav { top:0px!important; }
pi-shop.hu#$#body.aurora-nanobar-active { padding-top:0px!important; }
pi-shop.hu#$##firstLogNanobar { display:none!important; }
```

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -2024,6 +2024,9 @@ hardverapro.hu##.uad-details > .col-md-4.text-center > div[class*=" "]:has(> h4
 ! https://github.com/AdguardTeam/AdguardFilters/issues/31163
 figyelo.hu#$#div[class$="__interstitial_active"] { display: none!important; }
 figyelo.hu#$#div[class$="__interstitial_active"] + .page__body { display: block!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/36387
+pi-shop.hu#$##mobile-nav { top:0px!important; }
+pi-shop.hu#$#body.aurora-nanobar-active { padding-top:0px!important; }
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: That rule pack is meant for cookies as it is only broken when cookie message is removed.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt]: plz don't add styles without spaces

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt]: better `display: none !important;` instead of `display:none!important;`

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**